wiki/Chat_Console/en.md

```diff
@@ -1,275 +1,106 @@
+| ![Player is idle](Chat_Console-Idle.png "Player is idle")                                             | Dark Blue - Players is currently idle or not doing anything or just chatting.                                  |
+| ![Player is playing](Chat_Console-Playing.png "Player is playing")                                    | Grey - Playing a beatmap in solo.                                                                              |
+| ![Player is watching](Chat_Console-Watching.png "Player is watching")                                 | Light Blue - Watching a replay or spectating someone.                                                          |
+| ![Player is editing a beatmap](Chat_Console-Editing.png "Player is editing a beatmap")                | Red - Editing their own beatmap.                                                                               |
+| ![Player is testing a beatmap](Chat_Console-Testing.png "Player is testing a beatmap")                | Purple - Test playing a beatmap either it's their beatmap or not.                                              |
+| ![Player is submitting a beatmap](Chat_Console-Submitting.png "Player is submitting a beatmap")       | Turquoise - Submitting (either uploading or updating) the beatmap that they have made.                         |
+| ![Player is modding a beatmap](Chat_Console-Modding.png "Player is modding a beatmap")                | Green - Modding or editing someone else's beatmap.                                                             |
+| ![Player in Mulitplayer](Chat_Console-Multiplayer.png "Player in Mulitplayer")                        | Brown - User is in multiplayer, but not playing.                                                               |
+| ![Player is playing in Multiplayer](Chat_Console-Multiplaying.png "Player is playing in Multiplayer") | Yellow - Currently playing in multiplayer.                                                                     |
+| ![Player is Afk](Chat_Console-Afk.png "Player is Afk")                                                | Black - Inactive or away from keyboard (afk).                                                                  |
+| ![Player is connected via IRC](Chat_Console-IRC.png "Player is connected via IRC")                    | Dark Blue without content - Player is not ingame, but logged in from an IRC client or stats are not available. |
 
-</div>
 Clicking on any user panel brings up an options screen.
 
-<center>
-</center>
-Press the number or click the bar to activate:-
+![User panel options](Chat_Console-Userpaneloptions.jpg "User panel options")
+
+Press the number or click the bar to activate:
 
 1.  **Start Spectating**: If the user is playing a beatmap and you have the beatmap, you can watch them while they play. Your name will be in their Spectators list.
```

Comment:
the numbering will get off because of that button (invite to game)... convert to bullet list?